export.js

```diff
@@ -0,0 +1,61 @@
+const files = require("./lib/files")

```

Comment:
Maybe we should name this file to be `xls-to-json-converter`? I find using `export / import` confusing.